test/pure/List.test.mo

```diff
@@ -179,30 +179,30 @@ object unit : T.TestableItem<()> {
 
 let hugeList = List.repeat('Y', 100_000);
 
-let mapResult = Suite.suite(
+let mapResult = suite(
   "mapResult",
   [
-    Suite.test(
+    test(
```

Comment:
The goal is to migrate from `matchers` to the [`test`](https://mops.one/test) package (see `Random.test.mo`), although this should be fine for now.

Reply:
Yeah, but I don't grok those well enough yet.